### ISSUE TITLE
Rustpython release

### ISSUE THREAD

**Issue by windelbouwman** [rickyhan/pickitup#4]
Hi,

Recently, rustpython was released to crates.io (https://crates.io/crates/rustpython) . Would it be possible to use this version instead of the the forked one? What do you still miss in the original rustpython version (https://github.com/RustPython/RustPython)?

**Comment by 0b01** [comment]
Congrats on the release!

I see that the importlib is now bootstrapped. Previously I used a hack to [store source files in browser Local Storage](https://github.com/RustPython/RustPython/compare/master...rickyhan:master#diff-c07a832c613c9da629f7b5531ac65cdbR17). I would love to update to the released version.

Is there a way to import Python modules in browser?

**Comment by windelbouwman** [comment]
No, I think you cannot load python modules in the browser. Good to know about this usecase! Thanks, we'll have a look at it.
